### DISCUSSION TITLE
Welcome to MiaoYan Discussions!

### DISCUSSION THREAD

**Discussion by tw93** [tw93/MiaoYan#242] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Monkey-kong** [comment]
编辑框command+c没法复制内容，有点不方便，预览框又可以复制，帮看看这是bug么？

**Reply by tw93** [reply]
已经修复，预计最近发一个版本
